cpanfile

```diff
@@ -50,6 +50,7 @@ requires 'Mojo::Base';
 requires 'Mojo::ByteStream';
 requires 'Mojo::IOLoop';
 requires 'Mojo::JSON';
+requires 'Mojo::RabbitMQ::Client';
```

Comment:
It is already packaged: https://build.opensuse.org/package/show/home:dheidler/perl-Mojo-RabbitMQ-Client
But it requires `Net::AMQP` which is only in `devel:languages:perl:CPAN-N`. Should I do a SR for that package into `devel:languages:perl` as well?